property_table.py

```diff
@@ -2499,12 +2499,12 @@ class PropertyTable:
             True,
             0,
         ),
-        "concentration": (
-            "Concentration",
+        "concentration_unsoft": (
+            "ConcentrationUnsoftened",
             1,
             np.float32,
             "dimensionless",
-            "Halo concentration assuming an NFW profile.",
+            "Halo concentration assuming an NFW profile. No particle softening.",
```

Comment:
The softening explanations in the property tables are somewhat inconsistent in their presentation, e.g. `No particle softening`, `not accounting for softening length.`. Perhaps harmonise?

Reply:
[344dd9a](https://github.com/SWIFTSIM/SOAP/pull/93/commits/344dd9a053a1c5de4ecd50a90f308c5dd1de7aca)